tests/utils/constraints.txt

```diff
@@ -43,6 +43,7 @@ botocore >= 1.10.0, < 1.14 ; python_version < '2.7' # adds support for the follo
 botocore >= 1.10.0 ; python_version >= '2.7' # adds support for the following AWS services: secretsmanager, fms, and acm-pca
 setuptools < 45 ; python_version <= '2.7' # setuptools 45 and later require python 3.5 or later
 cffi >= 1.14.2, != 1.14.3 # Yanked version which older versions of pip will still install:
+psutil < 5.6.0 ; python_version < '3' # Version above 5.6.0 breaks on python 2.x. See https://github.com/ansible/ansible/pull/70667
```

Comment:
You know you can also have multiple of these lines, one for Python < 3 and one for Python <= 2.6?

Reply:
Inside docker , pip install psutil==5.6.0 works fine , but when we try to install it through constraint.txt, its erroring out.

Reply:
I spent some time trying to figure out what goes wrong with Python 2.6 in CentOS 6, and I think I found the problem. It's a bug in pip that is apparently caused by some other lines in the same file. I've created #2256 to fix it. Once that's merged, I'll re-trigger the tests here.

Reply:
This works fine when I try to run this locally inside docker container.
`- hosts: localhost
     tasks:
         - pip:
                name: psutil
                 extra_args: "-c <path>/constraints.txt"`
                 
I wonder if the constraints are being applied correctly . Can you please take a look @felixfontein  ?

Reply:
I'm not sure what your question is. With #2256 merged, the tests now pass.